playbooks/settings.yml

```diff
@@ -350,3 +354,57 @@ environments:
         networks:
           public: 5
         groups: [clients]
+
+  gpfs:
+    cpus: 2
+    memory: 1024
+    domain: sit.test
+
+    data:
+      # A zip file is necessary to install GPFS but it's not included with
+      # the ansible playbooks. You need to sign up with IBM to get access
+      # to the Storage Scale Developer edition. The environment variable
+      # STORAGE_SCALE_DEV_ZIP must be defined to point to the location of
+      # the downloaded file.
+      package: "{{ lookup('env', 'STORAGE_SCALE_DEV_ZIP') | default('') }}"
+
+      # This is the the regular expression used to identify the installation
+      # file inside the zip. It must have a capturing group to extract the
+      # version number.
+      pattern: "Storage_Scale_Developer-([0-9.]+)-x86_64-Linux-install"
+
+    nodes:
+      setup:
+        cpu_weight: 0
+        memory_weight: 2
+        os: centos8
+        networks:
+          private: 200
+          public: 200
+        ctdb:
+          public: 20
+        groups: [cluster, installer, gui, admin]
+        accounts: [default]
+
+      client:
+        cpu_weight: 1
+        memory_weight: 1
+        os: centos8
+        memory: 1024
```

Comment:
`gpfs.memory` above is already set as 1024. Do we still redefine it here?

Reply:
It's not necessary. I'll remove it.